Makefile

```diff
@@ -2,16 +2,10 @@
 SRCS := $(shell find . -type d -name 'vendor' -prune -o -name '*.go' -print)
 
 .PHONY: install
-install: glide #download dependencies (including test deps) for the package
-	glide install
-
-.PHONY: update
-update: glide #updates dependencies used by the package and installs them
-	glide update
-
-.PHONY: glide
-glide: # ensure the glide package tool is installed
-	which glide || go get github.com/Masterminds/glide
+install:
+	export GO111MODULE=on
+	git config --global url."git@github.com:lyft/".insteadOf "https://github.com/lyft/"
```

Comment:
Not sure this is applicable for anyone but at Lyft. Also not a fan of a makefile changing my global git config..

Reply:
already fixed. was not needed here